ZBaselines/Common/EntryPoints/core_ep-list.tsv

```diff
@@ -8,6 +8,7 @@ Models.BinaryClassificationEvaluator	Evaluates a binary classification scored da
 Models.BinaryCrossValidator	Cross validation for binary classification	Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro	CrossValidateBinary	Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro+Arguments	Microsoft.ML.Runtime.EntryPoints.CommonOutputs+MacroOutput`1[Microsoft.ML.Runtime.EntryPoints.CrossValidationBinaryMacro+Output]
 Models.ClassificationEvaluator	Evaluates a multi class classification scored dataset.	Microsoft.ML.Runtime.Data.Evaluate	MultiClass	Microsoft.ML.Runtime.Data.MultiClassMamlEvaluator+Arguments	Microsoft.ML.Runtime.EntryPoints.CommonOutputs+ClassificationEvaluateOutput
 Models.ClusterEvaluator	Evaluates a clustering scored dataset.	Microsoft.ML.Runtime.Data.Evaluate	Clustering	Microsoft.ML.Runtime.Data.ClusteringMamlEvaluator+Arguments	Microsoft.ML.Runtime.EntryPoints.CommonOutputs+CommonEvaluateOutput
+Models.CrossValidationResultsCombiner	Combine the metric data views returned from cross validation.	Microsoft.ML.Runtime.EntryPoints.CrossValidationMacro	CombineMetrics	Microsoft.ML.Runtime.EntryPoints.CrossValidationMacro+CombineMetricsInput	Microsoft.ML.Runtime.EntryPoints.CrossValidationMacro+CombinedOutput
```

Comment:
@yaeldekel  Seems like a positive change but out of curiosity what was the rationale behind it? I'll reference this change in my PR(not yet out) that augments CV and TrainTest macro to work with LearningPipeline framework.